### DISCUSSION TITLE
Copy button for notifications

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1290] [Feature Requests]
Would be great to have a "copy" button for notifications

**Comment by SJNBham** [comment]
Wonder if there's any benefit to integrating with Duplicate Posts plugin (3 million+ users) for this functionality.

https://wordpress.org/plugins/duplicate-post/

**Comment by stevejburge** [comment]
Thanks @SJNBham Hopefully this feature is simple enough we could include it in the PublishPress core

**Comment by andergmartins** [comment]
Related to #35

**Comment by stevejburge** [comment]
One more vote for this. They want to create multiple notifications for different post types.
